Example/Non-Card Payment Examples/Non-Card Payment Examples/SwishExampleViewController.swift

```diff
@@ -0,0 +1,107 @@
+//
```

Comment:
We should remove this step from adding a new PM: I don't think it's necessary for user-facing docs, and it's easier to build them in IntegrationTester for our own testing purposes